docs/docs/general/03-Best Practice/selecting-with-text.md

```diff
@@ -0,0 +1,173 @@
+# Selecting an Element with Text
```

Comment:
This is an awesome page. Do we have a blogpost already? It could definitely be converted to one!